backends/tfhe-cuda-backend/cuda/src/integer/scalar_comparison.cuh

```diff
@@ -813,9 +828,9 @@ __host__ void host_integer_radix_scalar_equality_check_kb(
       PANIC("Cuda error: integer operation not supported")
     }
 
-    host_compare_with_zero_equality<Torus>(msb_streams, gpu_indexes, gpu_count,
```

Comment:
If I read this right, we are removing `are_all_comparisons_block_true()` from inside `host_compare_blocks_with_zero()` to save ops when the latter is not need, as here and inside division. Is that right?

Reply:
Exactly